src/symmetricDifference.js

```diff
@@ -0,0 +1,19 @@
+export default function symmetricDifference(a, b){
+  if (Array.isArray(a) && Array.isArray(b)){
+    const symmetricDifference = [];
+    for(var i = 0; i<a.length; i++){
+      if(b.includes(a[i]) === false){
+        symmetricDifference.push(a[i]);
+      }
+    }
+    for(var i = 0; i<b.length; i++){
```

Comment:
Check spacing in this file, keep spacing consistent for readability. Specifically in your `for` loops.